docs/topics/gradle/gradle-configure-project.md

```diff
@@ -348,6 +348,24 @@ tasks.withType<UsesKotlinJavaToolchain>().configureEach {
 }
 ```
 
+### Lazy Kotlin/JVM task creation
+
+The Kotlin Gradle plugin registers all the tasks and doesn't configure them on a dry run.
```

Comment:
@Tapchicoma, in What's new, we wrote:

> For projects with the "org.jetbrains.kotlin.gradle.jvm" plugin on Gradle versions 7.3+, the Kotlin Gradle Plugin doesn’t create and configure the task "compileKotlin" eagerly anymore. On lower Gradle versions, it just registers all the tasks and doesn’t configure them on a dry run. The same behavior is now when using Gradle 7.3+.

But we keep docs up to date and usually don't write details about how it was in the past.
All I could add is:
`Starting from Kotlin 1.8.20, ...`

WDYT, is it worth adding?

Reply:
"KGP nuances" sounds very ambiguous to me. What about "Other details"?

Reply:
"Other details" sounds ok for me